addon/actions/triggerable.js

```diff
@@ -0,0 +1,96 @@
+import Ember from 'ember';
+import { simpleFindElementWithAssert, buildSelector, getContext } from '../helpers';
+
+function triggerableInternal(tree, event, selector, options, context) {
+  var fullSelector = buildSelector(tree, selector, options);
+
+  // Run this to validate if the element exists
+  simpleFindElementWithAssert(tree, fullSelector, options)
+
+  if (context) {
+    if (options.testContainer) {
+      Ember.$(fullSelector, options.testContainer).trigger(event);
+    } else {
+      context.$(fullSelector).trigger(event);
+    }
+  } else {
+    /* global triggerEvent */
+    triggerEvent(fullSelector, options.testContainer, event);
+  }
+}
+
+/**
+ *
+ * Focus on element matched by selector.
+ *
+ * @example
+ *
+ * // <input class="name">
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   focusOnName: triggerable('focus'. '.name')
+ * });
+ *
+ * // focuses on element with selector '.name'
+ * page.focusOnName();
+ *
+ * @example
+ *
+ * // <div class="scope">
+ * //   <input class="name">
+ * // </div>
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   focusOnName: triggerable('focus', '.name', { scope: '.scope' })
+ * });
+ *
+ * // focuses on element with selector '.scope .name'
+ * page.focusOnName();
+ *
+ * @example
+ *
+ * // <div class="scope">
+ * //   <input class="name">
+ * // </div>
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   scope: '.scope',
+ *   continue: triggerable('focus', '.name')
+ * });
+ *
+ * // clicks on element with selector '.scope button.continue'
+ * page.focusOnName();
+ *
+ * @public
+ *
+ * @param {string} event - Event to be triggered
```

Comment:
`Event to trigger`... I know, I'm not the one to correct english grammar so what I'm suggesting could be wrong